front/src/containers/Batch.js

```diff
@@ -468,6 +468,37 @@ class Batch extends React.Component {
 
   renderChat() {
     const {sendMessage, user, voteCasualForm, chat, batch, currentRound, vote} = this.props;
+
+
+    // DYNAMIC TEAM SIZE: Make the chat look different when they're in an individual round, so that users are less confused.
+    if(batch.dynamicTeamSize){
+      if(this.props.currentTeam){ //this starts out as null at the very beginning
+        if(this.props.currentTeam.length === 1){ // this means we're an individual.
+          //change background color to green
+          if(document.getElementsByClassName("card-body")[0]){
+            document.getElementsByClassName("card-body")[0].style.backgroundColor = "#529A88" //green
+          }
+          // NOTE: The way that this code is dynamically called makes it impossible to create variable names, hence some repetition.
```

Comment:
You could probably do this by setting a custom property on the original node, but this is a fine workaround.

Reply:
Ok!